Memoria.Launcher/ModManagerWindow.cs

```diff
@@ -378,7 +415,8 @@ private void DownloadEnd(Object sender, AsyncCompletedEventArgs e)
                 String downloadingModName = downloadingMod.Name;
                 String path = Mod.INSTALLATION_TMP + "/" + (downloadingMod.InstallationPath ?? downloadingModName);
                 Boolean success = false;
-                if (String.IsNullOrEmpty(downloadingMod.DownloadFormat) || downloadingMod.DownloadFormat == "Zip")
+                String downloadFormatLower = downloadingMod.DownloadFormat ?? "zip".ToLower();
```

Comment:
note for self this is a bug it's missing brackets should be `(downloadingMod.DownloadFormat ?? "zip").ToLower();`

Reply:
Resolved in [ac97eca](https://github.com/Albeoris/Memoria/pull/711/commits/ac97ecaeb0fe8f835ae1fe729c3883cd9592b377)